client/components/PetTypeTile.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const PetTypeTile = (props) => {
+    const { id, type, imageUrl, description } = props.typeOfPet
+
+    let descriptionBlock
+    if(description) {
+        descriptionBlock= <p>{description}</p>;
+    }
```

Comment:
not a requirement but a suggestion to possibly display filler text if there is no description
```
Description: N/A
```